presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -895,6 +895,22 @@ public void testView()
         assertUpdate("DROP VIEW test_view");
     }
 
+    @Test
+    public void testViewWithReservedKeywords()
+    {
+        skipTestUnless(supportsViews());
+
+        assertUpdate("CREATE TABLE \"group\" AS SELECT * FROM orders", "SELECT count(*) FROM orders");
+
+        @Language("SQL") String query = "SELECT orderkey, orderstatus, totalprice / 2 half FROM orders";
+
+        assertUpdate("CREATE OR REPLACE VIEW test_view AS " + "SELECT orderkey, orderstatus, totalprice / 2 half FROM \"group\"");
+
+        assertQuery("SELECT * FROM test_view", query);
+
+        assertUpdate("DROP VIEW test_view");
+    }
```

Comment:
nit: one of the review comments from the original PR also recommended running `SHOW CREATE VIEW` on the view created in this test, maybe we can add that as well.